tests/golos.referral_tests.cpp

```diff
@@ -111,6 +118,29 @@ BOOST_AUTO_TEST_SUITE(golos_referral_tests)
 
  } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(transfer_tests, golos_referral_tester) try {
```

Comment:
Please split checks for logical blocks and add some comment on them using `BOOST_TEST_MESSAGE`.
As example see https://github.com/GolosChain/golos-smart/blob/master/tests/golos.publication_tests.cpp#L210-L239. This will help in the future when parsing falls faster to understand what should happen.